topics/metabolomics/tutorials/msPurity_training/tutorial.md

```diff
@@ -0,0 +1,1006 @@
+The SQLite schema of the spectral database can be detailed here : 
+
+![Full schema of the database it will create ](../../images/tutorial_msms_DB_createDatabase.png "Full schema of the database created by this tool")
+
+Add more explanations about the schema  !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+> ### {% icon hands_on %} Hands-on : msPurity.createDatabase {% icon tool %}
+>
+> For our tutorial we will complete the following parameters for **msPurity.createDatabase {% icon tool %}** tool : 
+>   - **xcmsSet or CAMERA object** : the XCMS preprocessing RData file you used for msPurity.frag4feature {% icon tool %} tool and that outputed from the first part of the tutorial. It should be named `xset.merged.groupChromPeaks.RData`. And it hasn't been run through CAMERA processing.
+>  {: .text-justify}
+>   - **Use CAMERA object or XCMS object from RData?** : Just precise if your xcmsSet object has been run through CAMERA processing or not. For us, in our tutorial, it is just a xcmsSet object which has no CAMERA data. So, `keep it on XCMS (xset)`.
+>  {: .text-justify}
+>   - **purityA object (frag4feature output)** : the purityA object in RData file that outputed from msPurity.frag4feature {% icon tool %} tool. But if you also processed some filtering or averaging on it, it can also be the RData from any of these ones. So for this tutorial your RData file should be named `msPurity.averageSpectra(all)_on_your_data.RData`.
+>  {: .text-justify}
+>   - **Include EIC data?** : just answer here if you want to obtain EICs for each peak and store them in the database. If you say "yes" you have to know that it takes quite a lot of time to process it. For our tutorial you can `keep it to "no"`.
+>  {: .text-justify}
+>   - **Add a different grouped peaklist to database?** : if you want to add some grouped peaks. For our tutorial `keep it on user default`. But if you want to add something you need to create a dataframe before that is derived from the xset object of the first part of this tutorial. 
+>  {: .text-justify}
+>
+{: .hands_on}
+
+
+## 7 - (Optionnal) Matching of databases
+  
+Firstly, this function is coupled and must be executed after the previous **msPurity.createDatabase {% icon tool %}** tool. Indeed, this **msPurity.spectralMatching {% icon tool %}** tool allows the matching between the just created database and an other one.  
+
+So, the first one is a query SQLite database which should contain the *"unknown"* spectral database generated by the **msPurity.createDatabase {% icon tool %}** tool as part of the data processing workflow.  
+The second database is the spectral database of the library must contain the *“known”* spectra coming from public resources or generated by the users. The SQLite library database defaults to MoNA data, including Massbank, HMDB, LipidBlast and GNPS. A larger database can be downloaded from here??????????.
```

Comment:
Have to check the link to have a larger database. It wasn't good when I checked...